pkg/manager/member/tidb_init_manager.go

```diff
@@ -463,9 +466,10 @@ func getInitMeta(ti *v1alpha1.TidbInitializer) (metav1.ObjectMeta, label.Label)
 	initLabel := label.NewInitializer().Instance(ti.Name).Initializer(ti.Name)
 
 	objMeta := metav1.ObjectMeta{
-		Name:      name,
-		Namespace: ti.Namespace,
-		Labels:    initLabel,
+		Name:        name,
+		Namespace:   ti.Namespace,
+		Labels:      initLabel,
```

Comment:
This does not include customized labels.

Reply:
https://github.com/pingcap/tidb-operator/pull/4029/commits/3ea7b432dc9678acd1f63de8d00b47553b4ddf7a